config/initializers/inflections.rb

```diff
@@ -1,16 +0,0 @@
-# Be sure to restart your server when you modify this file.
```

Comment:
This file should stay - even though it is empty, it is a placeholder for if inflectors are required - an experienced ruby developer will just recreate the file but someone newer to the language might not realise they should. Same logic as using `.gitkeep` files